### PR TITLE
Hivemind no longer has heals, and can no longer build walls

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request

Removes hivemind heal/wall making threw walls.
## Why It's Good For The Game
This is incredibly annoying for marines and basically doesn't make sense considering this is supposed to be an AI counterpart, and the marine AI can't heal/build walls on will. Why should the hivemind be able to? This is incredibly annoying to play against as marine. It can still secrete resin however, and can still use pheromones.

## Changelog
:cl:
balance: Hivemind can no longer heal or build walls
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
Drop the building to Elder then.

Also up the cooldown on building a decent amount, it's kinda low.

**Comment by RenGusta** [comment]
Or let an AI control drones and nano swarms to simulate heals and remote support

**Comment by Kuro-ai** [comment]
> Drop the building to Elder then.
> 
> Also up the cooldown on building a decent amount, it's kinda low.

A invisible force that cant be reacted to cant have building capacity period

**Comment by jroinc** [comment]
> Or let an AI control drones and nano swarms to simulate heals and remote support

You know, I'd actually be fine with that. Less the heals, cause medics/selfmed, but the barricadedrones would be neat.

> A invisible force that cant be reacted to cant have building capacity period

Eh, I'm fine with it. You can 100% REACT to it by breaking the walls, doofus :P

More seriously, give it a ~4-5 tile radius around marines that it can't enter. It can still stay still and not be forced out, but then you're just locked it down. Also, give it the boiler glow.

**Comment by Kuro-ai** [comment]
> > Or let an AI control drones and nano swarms to simulate heals and remote support
> 
> You know, I'd actually be fine with that. Less the heals, cause medics/selfmed, but the barricadedrones would be neat.
> 
> > A invisible force that cant be reacted to cant have building capacity period
> 
> Eh, I'm fine with it. You can 100% REACT to it by breaking the walls, doofus :P
> 
> More seriously, give it a ~4-5 tile radius around marines that it can't enter. It can still stay still and not be forced out, but then you're just locked it down. Also, give it the boiler glow.

You are,but a lot of people aren't,therefore to the drain it goes

**Comment by terranaut1** [comment]
Hivemind has zero kill or capture potential while locking up an entire xeno slot for the balance calculation, "it's annoying for the enemy team" is really a garbage argument to nerf something into complete uselessness, you can 100% react to it by destroying shit it plants or killing the core or trapping it in a weed field somewhere. If you think it builds way too much then up the cooldowns or plasma cost a bit but this trend of just nerfing every single vaguely good thing the aliens have while constantly buffing marines is really fucking tiring.

**Comment by psykzz** [comment]
Hivemind already has slower building and slower weeds. I'd rather we tune those values other than removing the caste.

I'd rather add a vision check that if a marine is in view range the hivemind can't build.

**Comment by XElectricX** [comment]
For this to be fair, then make the hivemind not take a larva slot.

**Comment by davidchan13** [comment]
Main issues with hivemind building is that its impossible to tell that something is being built in that spot and it doesn't appear that the building cares if weeds are present or not (I've seen walls being dropped behind cades where weeds can't reach)

If Marines/Xenos could actually see that something was going in that tile and even prevent it by destroying the weeds, I don't think the hivemind would be as annoying for marines to deal with as is.

I am fully against the heal being taken away, hivemind needs more abilities, not less. As it stands without the heal hivemind is just a mobile pheromone dispenser and that's not really worth losing a drone for.

**Comment by MerryweatherLost** [comment]
**Overview**
The changes requested would make the hivemind, more or less useless. The building should be changed or tuned yes, and possibly the health as well, but removing these outright would make the hivemind what I stated before.

> I'd rather add a vision check that if a marine is in view range the hivemind can't build.

I agree with this statement.

The heals should continue to be around, but it can be tuned along with the other features the hivemind has, not torn out.

File: code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm [hunk 1]
```diff
@@ -67,12 +68,6 @@
 	// *** Evolution *** //
 	upgrade_threshold = 500
 
-	// *** Abilities *** //
-	actions = list(
-		/datum/action/xeno_action/plant_weeds/slow,
-		/datum/action/xeno_action/activable/psychic_cure,
-		)
-
```
Comment (MerryweatherLost) [inline code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm:75]: Readd these.

File: code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm [hunk 2]
```diff
@@ -88,13 +83,6 @@
 	// *** Evolution *** //
 	upgrade_threshold = 700
 
-	// *** Abilities *** //
-	actions = list(
-		/datum/action/xeno_action/plant_weeds/slow,
-		/datum/action/xeno_action/activable/psychic_cure,
-		/datum/action/xeno_action/toggle_pheromones,
-		)
-
```
Comment (MerryweatherLost) [inline code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm:97]: These as well.

File: code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm [hunk 3]
```diff
@@ -110,11 +98,4 @@
...
 	upgrade_threshold = 1000
 
-	// *** Abilities *** //
-	actions = list(
-		/datum/action/xeno_action/plant_weeds/slow,
-		/datum/action/xeno_action/activable/psychic_cure,
-		/datum/action/xeno_action/choose_resin,
-		/datum/action/xeno_action/toggle_pheromones,
-		/datum/action/xeno_action/activable/secrete_resin/slow,
-		)
```
Comment (MerryweatherLost) [inline code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm:120]: And those. I'll dm you a solution to the psychic cure which can nerf them but not remove them.

**Pull request closed without merging**